Mailbox.Tests/DataLoaderTests.cs

```diff
@@ -0,0 +1,69 @@
+﻿using Microsoft.VisualStudio.TestTools.UnitTesting;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Text;
+
+namespace Mailbox.Tests
+{
+    [TestClass]
+    public class DataLoaderTests
+    {
+
+        [TestMethod]
+        [ExpectedException(typeof(ArgumentNullException))]
+        public void ConstructorPassedNull()
+        {
+            DataLoader dataLoader = new DataLoader(null);
+            dataLoader.Dispose();
+        }
+
+        [TestMethod]
+        [ExpectedException(typeof(ArgumentNullException))]
+        public void SavePassedNull()
+        {
+            MemoryStream stream = new MemoryStream();
+            DataLoader dataLoader = new DataLoader(stream);
+
+            dataLoader.Save(null);
+            dataLoader.Dispose();
+        }
+
+        [TestMethod]
+        public void LoadSaveSucessful()
+        {
+            MemoryStream stream = new MemoryStream();
+            DataLoader dataLoader = new DataLoader(stream);
+            List<Mailbox> mailboxes = getTestData();
+
+            dataLoader.Save(mailboxes);
+            List<Mailbox> loadedMailboxes = dataLoader.Load();
+
+            for(int i = 0; i < loadedMailboxes.Count; i++)
+            {
+                Assert.AreEqual(mailboxes[i].Owner, loadedMailboxes[i].Owner);
+                Assert.AreEqual(mailboxes[i].Location, loadedMailboxes[i].Location);
+                Assert.AreEqual(mailboxes[i]._Size, loadedMailboxes[i]._Size);
+            }
+            dataLoader.Dispose();
+        }
+
+        public List<Mailbox> getTestData()
+        {
+            List<Mailbox> mailboxes = new List<Mailbox>();
+
+            //Locations for the mailboxes
+            ValueTuple<int, int> loc = (0, 0);
```

Comment:
ValueTuple syntax can be simplified
```suggestion
           (int, int) loc = (0, 0);
```